core/file/dicom/mapper.cpp

```diff
@@ -35,6 +35,23 @@ namespace MR {
 
       std::unique_ptr<MR::ImageIO::Base> dicom_to_mapper (MR::Header& H, vector<std::shared_ptr<Series>>& series)
       {
+        //ENVVAR name: MRTRIX_PRESERVE_PHILIPS_ISO
```

Comment:
Could a config file entry be more appropriate for this? Still possible to modify on a per-invocation basus using `-config` command-line option.

Reply:
I thought about this, but I don't expect anyone would ever want to set this as the default in the installation. I don't think this will ever be used for anything other than debugging on the odd occasion where someone isn't sure about what's happened...

Reply:
:+1: